tasks.c

```diff
@@ -938,7 +938,9 @@ static void prvYieldForTask( TCB_t * pxTCB,
             {
                 if( xDecrementTopPriority != pdFALSE )
                 {
-                    uxTopReadyPriority--;
+                    if(uxTopReadyPriority > 0) {
+                        uxTopReadyPriority--;    
+                    }
```

Comment:
Nit pick, non-blocking: I realize `0` isn't exactly a magic number in this context BUT I would prefer this line to say `uxTopReadyPriority > tskIDLE_PRIORITY`. This would read like "The Top Ready Priority cannot be less than the idle task" rather than "The Top Ready Priority cannot be less than 0". While these are functionally the same, pinning this to the idle task would make it clearer that no task can be lower in priority than the idle task.